store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/util/MergeElementFunction.java

```diff
@@ -77,16 +91,37 @@ public ApplyViewToElementsFunction(final Map<String, Object> context) throws Gaf
             }
             // Validate the supplied context before using
             validate(context);
+
+            updateViewWithValidationFromSchema(context);
+
             this.context = Collections.unmodifiableMap(context);
         } catch (final Exception e) {
             throw new GafferCheckedException("Unable to create TemporaryResultsGraph", e);
```

Comment:
Do we need to throw an exception here? Can we not just log the issue?

Reply:
If the result graph is not able to be created for merging, then the currently execution cannot continue.
Except get thrown and more info is added further up the stack.
There are places higher in the Stack that will log this error, but with more information.